Code/GraphMol/MolStandardize/MolStandardize.cpp

```diff
@@ -285,5 +285,25 @@ std::vector<std::string> enumerateTautomerSmiles(
   return res.smiles();
 }
 
+void disconnectOrganometallics(RWMol &mol) {
+  RDKit::MolStandardize::MetalDisconnectorOptions mdOptions;
+  mdOptions.splitGrignards = true;
```

Comment:
I think this function should use the defaults for `MetalDisconnectorOptions`. If that means that the defaults need to be changed to the explicit values you have here, then please do so.

Reply:
That seems like a good solution.

Reply:
Great.  That was the last thing that I wasn't sure about.  I'm running tests locally and then it should be good to push.